### PR TITLE
Update proxies_v8.ts

### PR DESCRIPTION
**Pull request by dkttarento**
proxies_v8  changes for top content microsites.

### CODE REVIEW

File: src/proxies_v8/proxies_v8.ts [hunk 1]
```diff
@@ -267,7 +267,7 @@ proxiesV8.use('/microsite/read/insights',
 
 proxiesV8.use('/content/aggregation/search',
   // tslint:disable-next-line: max-line-length
-  proxyCreatorSunbirdSearch(express.Router(), `${CONSTANTS.KONG_API_BASE}/content/aggregation/search`)
+  proxyCreatorSunbirdSearch(express.Router(), `${CONSTANTS.KONG_API_BASE}/aggregation/search`)
```
Comment (karthik-tarento) [inline src/proxies_v8/proxies_v8.ts:270]: I don't see any `${CONSTANTS.KONG_API_BASE}/aggregation/search -- which service it points too ?
`${CONSTANTS.KONG_API_BASE}/aggregation/search -- is very generic -- what is this regarding too...

**Review by karthik-tarento — CHANGES_REQUESTED** [review]

File: src/proxies_v8/proxies_v8.ts (hunk 1 above)
Comment (dkttarento) [reply src/proxies_v8/proxies_v8.ts:270]: @karthik-tarento , This is regarding Top courses - Microsites , It is pointing to cb- ext service.,

Comment (Sahil-tarento) [reply src/proxies_v8/proxies_v8.ts:270]: @karthik-tarento  This is related to top courses @dkttarento please made this url changes as understood what this is related to

Comment (dkttarento) [reply src/proxies_v8/proxies_v8.ts:270]: @karthik-tarento , This is related to Top courses microsite, as with "/content/aggregation/search", was getting 502 Bad gateway, as discussed with @Sahil-tarento , changes are made to "/aggregation/search".

**Pull request closed without merging**